components/Player/PlayerTabs.jsx

```diff
@@ -2,9 +2,9 @@ import React from 'react';
 import TabBar from '../TabBar';
 import { playerPages } from '../Header/Pages';
 
-const playerPagesMapped = (accountId) => playerPages.map((e) => Object.assign({}, e, {
-  route: `/players/${accountId}/${e.name.toLowerCase()}`,
-  label: e.name,
+const playerPagesMapped = (accountId) => playerPages.map(page => ({
+  route: `/players/${accountId}/${page.name.toLowerCase()}`,
```

Comment:
CI is passing.  It's probably the local IDE not recognizing the spread operator (one of the reasons I still use object.assign)